### PR TITLE
Update header to 2022

### PR DESCRIPTION
**Pull request by y0urself**
**What**:

**Why**:

**How**:

**Checklist**:

- [ ] Tests
- [ ] PR merge commit message adjusted

### CODE REVIEW

**Review by Kraemii — COMMENTED** [review]
There are still 5 files with the year 2021:
`./CMakeLists.txt`
`./cmake/GetGit.cmake`
`./doc/CmakeLists.txt`
`./nasl/nasl_builtin_find_service.c`
`./nasl/nasl_text_utils.c`

In `./nasl/nasl.c` in line 248 and in `./src/openvas.c` in line 529 the old year 2021 is printed.

File: nasl/capture_packet.h [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (Kraemii) [inline nasl/capture_packet.h:1]: ?

File: nasl/tests/Makefile [hunk 2]
```diff
@@ -1,21 +1,3 @@
...
-#
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with this program; if not, write to the Free Software
-# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
-
```
Comment (Kraemii) [inline nasl/tests/Makefile:18]: Why is the header removed here and added to the `README.txt` file?

**Review by Kraemii — DISMISSED** [review]
lgtm

**Review by jjnicola — APPROVED** [review]

**Review by Kraemii — APPROVED** [review]

**Pull request merged**